mongodbatlas/data_source_mongodbatlas_clusters.go

```diff
@@ -350,37 +349,41 @@ func flattenClusters(ctx context.Context, d *schema.ResourceData, conn *matlas.C
-			"provider_disk_type_name":      clusters[i].ProviderSettings.DiskTypeName,
-			"provider_encrypt_ebs_volume":  clusters[i].ProviderSettings.EncryptEBSVolume,
-			"provider_instance_size_name":  clusters[i].ProviderSettings.InstanceSizeName,
-			"provider_name":                clusters[i].ProviderSettings.ProviderName,
-			"provider_region_name":         clusters[i].ProviderSettings.RegionName,
-			"bi_connector":                 flattenBiConnector(clusters[i].BiConnector),
-			"bi_connector_config":          flattenBiConnectorConfig(clusters[i].BiConnector),
-			"replication_specs":            flattenReplicationSpecs(clusters[i].ReplicationSpecs),
-			"labels":                       flattenLabels(clusters[i].Labels),
-			"snapshot_backup_policy":       snapshotBackupPolicy,
+			"auto_scaling_disk_gb_enabled":            clusters[i].BackupEnabled,
+			"backup_enabled":                          clusters[i].BackupEnabled,
+			"provider_backup_enabled":                 clusters[i].ProviderBackupEnabled,
+			"cluster_type":                            clusters[i].ClusterType,
+			"connection_strings":                      flattenConnectionStrings(clusters[i].ConnectionStrings),
+			"disk_size_gb":                            clusters[i].DiskSizeGB,
+			"encryption_at_rest_provider":             clusters[i].EncryptionAtRestProvider,
+			"mongo_db_major_version":                  clusters[i].MongoDBMajorVersion,
+			"name":                                    clusters[i].Name,
+			"num_shards":                              clusters[i].NumShards,
+			"mongo_db_version":                        clusters[i].MongoDBVersion,
+			"mongo_uri":                               clusters[i].MongoURI,
+			"mongo_uri_updated":                       clusters[i].MongoURIUpdated,
+			"mongo_uri_with_options":                  clusters[i].MongoURIWithOptions,
+			"pit_enabled":                             clusters[i].PitEnabled,
+			"paused":                                  clusters[i].Paused,
+			"srv_address":                             clusters[i].SrvAddress,
+			"state_name":                              clusters[i].StateName,
+			"replication_factor":                      clusters[i].ReplicationFactor,
+			"backing_provider_name":                   clusters[i].ProviderSettings.BackingProviderName,
+			"provider_disk_iops":                      clusters[i].ProviderSettings.DiskIOPS,
+			"provider_disk_type_name":                 clusters[i].ProviderSettings.DiskTypeName,
+			"provider_encrypt_ebs_volume":             clusters[i].ProviderSettings.EncryptEBSVolume,
+			"provider_instance_size_name":             clusters[i].ProviderSettings.InstanceSizeName,
+			"provider_name":                           clusters[i].ProviderSettings.ProviderName,
+			"provider_region_name":                    clusters[i].ProviderSettings.RegionName,
+			"bi_connector":                            flattenBiConnector(clusters[i].BiConnector),
```

Comment:
Nitpick:

Seen this one is pretty long, can we do two things here? First group all the composite fields together, and then order everything alphabetically because it is kinda hard to read here

Reply:
changed